content/security.md

```diff
@@ -34,90 +35,91 @@ Without DNSSEC, a malicious hacker can point SimpleLogin MX record to their own
 
 CAA provides a way for domain holders to reduce that risk. Without CAA, someone could potentially obtain an unauthorized SSL certificate for SimpleLogin domains that could allow man-in-the-middle hacks.
 
-All SimpleLogin certificates are issued by [Letsencrypt](https://letsencrypt.org).
+All SimpleLogin certificates are issued by [Letsencrypt](https://letsencrypt.org). Witch is one of the most trusted CA's in the world.
```

Comment:
Witch -> which. Whether Letsencrypt is one of the most trusted one is debatable (i.e. not a fact) so I wouldn't put this sentence here.

Reply:
Made a spelling mistake. Which is fine. And most people i talk to seem to think so? Everybody only wants to use them. IF you don't trust them, why use them for your service?

Reply:
Changed it to "Which is a certificate authority SipleLogin trusts". What do you thin? :)